main.tf

```diff
@@ -2,7 +2,28 @@ resource "google_pubsub_topic" "topic" {
   name = "hedwig-${var.topic}"
 }
 
+resource "google_pubsub_subscription" "firehose" {
+  count = var.enable_firehose_all_messages ? 1 : 0
+  name  = "hedwig-${var.topic}-firehose"
+  topic = "hedwig-${var.topic}"
+
+  cloud_storage_config {
+    bucket          = var.firehose_bucket
+    filename_prefix = local.firehose_prefix
```

Comment:
Q.  What will be the final name of the files? 
Context: I see there's a `filename_suffix` as well, which I figure is for denfining filenames but I am not sure what the end result would look like. I assume it will create multiple files for each subscription.

Reply:
Nvm, found the info : https://cloud.google.com/pubsub/docs/create-cloudstorage-subscription#file_names

Reply:
That's right and I added the _`filename_suffix`_ setting to help with tasks like adding an _`.avro`_ suffix to filenames.